qiskit_experiments/randomized_benchmarking/clifford_utils.py

```diff
@@ -0,0 +1,216 @@
+        where `num` is between 0 and 11519.
+        """
+        return Clifford(self.clifford_2_qubit_circuit(num))
+
+    def random_cliffords(
+        self, num_qubits: int, size: int = 1, rng: Optional[Union[int, Generator]] = None
+    ):
+        """Generate a list of random clifford elements"""
+        if num_qubits > 2:
+            return random_clifford(num_qubits, seed=rng)
+
+        if rng is None:
+            rng = default_rng()
+
+        if isinstance(rng, int):
+            rng = default_rng(rng)
+
+        if num_qubits == 1:
+            samples = rng.integers(24, size=size)
+            return [Clifford(self.clifford_1_qubit_circuit(i)) for i in samples]
+        else:
+            samples = rng.integers(11520, size=size)
+            return [Clifford(self.clifford_2_qubit_circuit(i)) for i in samples]
+
+    def random_clifford_circuits(
+        self, num_qubits: int, size: int = 1, rng: Optional[Union[int, Generator]] = None
+    ):
+        """Generate a list of random clifford circuits"""
+        if num_qubits > 2:
+            return [random_clifford(num_qubits, seed=rng).to_circuit() for _ in range(size)]
+
+        if rng is None:
+            rng = default_rng()
+
+        if isinstance(rng, int):
+            rng = default_rng(rng)
+
+        if num_qubits == 1:
+            samples = rng.integers(24, size=size)
+            return [self.clifford_1_qubit_circuit(i) for i in samples]
+        else:
+            samples = rng.integers(11520, size=size)
+            return [self.clifford_2_qubit_circuit(i) for i in samples]
+
+    @lru_cache(maxsize=24)
+    def clifford_1_qubit_circuit(self, num):
+        """Return the 1-qubit clifford circuit corresponding to `num`
+        where `num` is between 0 and 23.
+        """
+        # pylint: disable=unbalanced-tuple-unpacking
+        # This is safe since `_unpack_num` returns list the size of the sig
+        (i, j, p) = self._unpack_num(num, self.CLIFFORD_1_QUBIT_SIG)
+        qc = QuantumCircuit(1)
+        if i == 1:
+            qc.h(0)
+        if j == 1:
+            qc.v(0)
+        if j == 2:
+            qc.v(0)
+            qc.v(0)
+        if p == 1:
+            qc.x(0)
+        if p == 2:
+            qc.y(0)
+        if p == 3:
+            qc.z(0)
+        return qc
+
+    @lru_cache(maxsize=11520)
+    def clifford_2_qubit_circuit(self, num):
+        """Return the 2-qubit clifford circuit corresponding to `num`
+        where `num` is between 0 and 11519.
+        """
+        vals = self._unpack_num_multi_sigs(num, self.CLIFFORD_2_QUBIT_SIGS)
+        qc = QuantumCircuit(2)
+        if vals[0] == 0 or vals[0] == 3:
+            (form, i0, i1, j0, j1, p0, p1) = vals
+        else:
+            (form, i0, i1, j0, j1, k0, k1, p0, p1) = vals
+        if i0 == 1:
+            qc.h(0)
+        if i1 == 1:
+            qc.h(1)
+        if j0 == 1:
+            qc.v(0)
+        if j0 == 2:
+            qc.v(0)
+            qc.v(0)
```

Comment:
replace 2 v's by w
another reductions (to reduce number of H's):
- HV = H Sdg H = S H S
- HW = S